about/complying_with_licenses.rst

```diff
@@ -58,31 +58,9 @@ This text reads as follows:
 Inclusion
 ---------
 
-The license does not specify how it has to be included, so anything is valid as
+The license does not specify how it has to be included, as
```

Comment:
The meaning of this sentence has changed. Previously it said that "anything was valid" if the condition is met. Now it says that "the licence does not specify how it has to be included" if the condition is met. Which is clearly incorrect. 

The licence doesn't specify how it has to be included, therefore users are free to do anything they choose as long as the licence can be displayed under some condition

Reply:
Ah yes, I see what you mean. How about:

The license text must be made available to the user. The license doesn't specify how it has to be included, but here are the most common approaches (you only need to implement one of them, not all).